code/modules/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -0,0 +1,227 @@
+						first_open = FALSE
+					if(untouched == TRUE)
+						new /obj/effect/decal/remains/human(loc)
+						untouched = FALSE
+					return 1
+				return 1
+			else
+				to_chat(user, "<span class='notice'>You can't dig up a grave with \the [S.name].</span>")
+				return 1
+		else
+			to_chat(user, "<span class='notice'>The grave has already been dug up.</span>")
+			return 1
+
+	else if(user.a_intent != INTENT_HELP) //checks to attempt to remove the grave entirely.
+		if(opened == TRUE)
+			if(istype(S,cutting_tool) && S.tool_behaviour == TOOL_SHOVEL)
+				to_chat(user, "<span class='notice'>You start to remove \the [src]  with \the [S].</span>")
+				if (do_after(user,15, target = src))
+					to_chat(user, "<span class='notice'>You remove \the [src]  completely.</span>")
+					SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+					deconstruct(TRUE)
+					return 1
+	return
+
+/obj/structure/closet/crate/grave/bust_open()
+	..()
+	opened = TRUE
+	update_icon()
+	dump_contents()
+	return
+
+/obj/structure/closet/crate/grave/lead_researcher
+	name = "ominous burial mound"
+	desc = "Even in a place filled to the brim with graves, this one shows a level of preperation and planning that fills you with dread."
+	icon = 'icons/obj/crates.dmi'
+	icon_state = "grave_lead"
+	lead_tomb = TRUE
+	first_open = TRUE
+
+/obj/structure/closet/crate/grave/lead_researcher/PopulateContents()  //ADVANCED GRAVEROBBING
+	..()
+	new /obj/effect/decal/remains/human(src)
+	new /obj/effect/decal/cleanable/blood/gibs/old(src)
+	new /obj/item/book/granter/crafting_recipe/boneyard_notes(src)
+
+/obj/item/book/granter/crafting_recipe/boneyard_notes
+	name = "The Complete Works of Lavaland Bone Architecture"
+	desc = "Pried from the lead archeologist's cold, dead hands, this seems to explain how ancient bone architecture was erected, long ago."
+	crafting_recipe_types = list(/datum/crafting_recipe/rib, /datum/crafting_recipe/boneshovel, /datum/crafting_recipe/halfskull, /datum/crafting_recipe/skull)
+	icon = 'icons/obj/library.dmi'
+	icon_state = "boneworking_learing"
+	oneuse = FALSE
+	remarks = list("Who knew you could bend bones that far back?", "I guess that was much easier before the planet heated up...", "So that's how they made those ruins survive the ashstorms. Neat!", "The page is just filled with insane ramblings about some 'legion' thing.", "But why would they need vinegar to polish the bones? And rags too?", "You spend a few moments cleaning dirt and blood off of the page, yeesh.")
+
+
+//***Fluff items for lore/intrigue
+/obj/item/paper/crumpled/muddy/fluff/elephant_graveyard
+	name = "Posted Warning"
+	desc = "It seems to be smudged with mud and... oil?"
+	info = "<B>ATTENTION TO WHOOM IT MAY CONCERN</B><BR><BR>This area is property of Nanotrasen Mining Devision.<BR><BR>Tresspassing in this area is illegal, highly dangerous, and subject to several NDAs.<br><br>Please turn back now, under intergalactic law section 48-R."
```

Comment:
Remove `ATTENTION`
`WHOOM` -> `WHOM`
`This area is property of Nanotrasen...` -> `This area is property of the Nanotrasen...`
`Devision` -> `Division`
`Tresspassing` -> `Trespassing`